pkg/controller/flink/client/api.go

```diff
@@ -231,7 +231,11 @@ func (c *FlinkJobManagerClient) SubmitJob(ctx context.Context, url string, jarID
 			// Flink returns a 500 when the entry class doesn't exist or crashes on start, but we want to fail fast
 			// in those cases
 			body := response.String()
-			if strings.Contains(body, "org.apache.flink.client.program.ProgramInvocationException") {
+			// ProgramInvocationException is thrown when the entry class doesn't exist or throws an exception
+			if strings.Contains(body, "org.apache.flink.client.program.ProgramInvocationException") ||
```

Comment:
Nit: Can you move "org.apache.flink.client.program.ProgramInvocationException" to a const err variable ?

Reply:
sure